pkg/cloudprovider/provider/anexia/provider.go

```diff
@@ -157,10 +160,19 @@ func provisionVM(ctx context.Context, client anxclient.Client) error {
 			VLAN:    config.VlanID,
 		}}
 
+		templateID := config.TemplateID
+		if templateID == "" {
```

Comment:
please add a comment for not checking if `config.Template` is set due to that already being checked below when reading the config

Reply:
:+1: